diambra/arena/engine/interface.py

```diff
@@ -17,124 +17,173 @@ def __init__(self, env_address):
         self.channel = grpc.insecure_channel(env_address)
         self.stub = interface_pb2_grpc.EnvServerStub(self.channel)
 
+        # Wait for grpc server to be ready
+        print("Trying to connect to DIAMBRA Engine server ...")
```

Comment:
Can you use a proper logger for these things, eg https://docs.python.org/3/library/logging.html?